torch/csrc/jit/codegen/cuda/dispatch.h

```diff
@@ -445,6 +451,9 @@ class TORCH_CUDA_API OptInDispatch {
   virtual void handle(kir::Allocate*) {
     AT_ERROR("Handle not overriden for Allocate.");
   }
+  virtual void handle(kir::Sync*) {
+    AT_ERROR("Handle not overriden for Sync.");
```

Comment:
nit: I know there's a bit of inconsistency, but most checks use TORCH_INTERNAL_ASSERT(false, "message")

Reply:
I believe in the definition of AT_ERROR it mentions a preference for `TORCH_INTERNAL_ASSERT(false,` or `TORCH_CHECK(false,`

Reply:
I went ahead and changed all the AT_ERROR to TORCH_INTERNAL_ASSERT.